x-pack/plugins/security/public/management/api_keys/api_keys_grid/api_key_flyout.tsx

```diff
@@ -251,243 +296,440 @@ export const ApiKeyFlyout: FunctionComponent<ApiKeyFlyoutProps> = ({
+                              defaultMessage="Owner"
+                            />
+                          </strong>
+                        </EuiText>
+                      </EuiFlexItem>
+                      <EuiFlexItem grow={false}>
+                        <EuiFlexGroup justifyContent="flexEnd" alignItems="center" gutterSize="xs">
+                          <EuiFlexItem grow={false}>
+                            <EuiIcon type="user" />
+                          </EuiFlexItem>
+                          <EuiFlexItem grow={false}>{apiKey.username}</EuiFlexItem>
+                        </EuiFlexGroup>
+                      </EuiFlexItem>
+                    </EuiFlexGroup>
+                    <EuiHorizontalRule margin="s" />
+
+                    <EuiFlexGroup alignItems="center" justifyContent="spaceBetween">
+                      <EuiFlexItem>
+                        <EuiText size="xs">
+                          <strong>
+                            <FormattedMessage
+                              id="xpack.security.accountManagement.apiKeyFlyout.typeLabel"
+                              defaultMessage="Type"
+                            />
+                          </strong>
+                        </EuiText>
+                      </EuiFlexItem>
+                      <EuiFlexItem grow={false}>
                         <ApiKeyBadge type={apiKey.type} />
-                      </EuiFormRow>
-                    </EuiFlexItem>
-                    <EuiFlexItem grow={false}>
-                      <EuiFormRow
-                        label={
-                          <FormattedMessage
-                            id="xpack.security.accountManagement.apiKeyFlyout.ownerLabel"
-                            defaultMessage="Owner"
-                          />
-                        }
-                      >
-                        <UsernameWithIcon username={apiKey.username} />
-                      </EuiFormRow>
-                    </EuiFlexItem>
-                    <EuiFlexItem grow={false}>
-                      <EuiFormRow
-                        label={
-                          <FormattedMessage
-                            id="xpack.security.accountManagement.apiKeyFlyout.createdLabel"
-                            defaultMessage="Created"
-                          />
-                        }
-                      >
+                      </EuiFlexItem>
+                    </EuiFlexGroup>
+                    <EuiHorizontalRule margin="s" />
+
+                    <EuiFlexGroup alignItems="center" justifyContent="spaceBetween">
+                      <EuiFlexItem>
+                        <EuiText size="xs">
+                          <strong>
+                            <FormattedMessage
+                              id="xpack.security.accountManagement.apiKeyFlyout.realmLabel"
+                              defaultMessage="Realm"
```

Comment:
I don't have a strong opinion here, I can think of some use cases where it's useful to know this information but that's really a design question @MichaelMarcialis?

Reply:
I reviewed this with @legrego, and we're in favor of removing realm from the UI here. Displaying the realm doesn't really add value to the end user here beyond what the other information provides, and it is queryable via API if absolutely needed.

Reply:
I'm fine with just removing it but will wait for either @julianrosado or @MichaelMarcialis to chime in with whether they think it's very important.

Reply:
I can confidently say I am unfamiliar enough with the importance of showing realm in the API key info panel that I would defer to you fine gentlemen on what to do with it. 

I definitely pulled that 1-to-1 from an existing API key view we had in Kibana _at some point_, so presumably there was a reason to include it when that was made.

It doesn't impact the design either way really, so the question is its inclusion more likely to be a source of confusion (remove it) or a niche helpful tidbit of info (keep it)?

Reply:
Alright, removed Realm in latest update, which should make this PR ready for final review/merge :)

Reply:
> is its inclusion more likely to be a source of confusion (remove it) or a niche helpful tidbit of info (keep it)

I just reviewed with the team, and the consensus is to remove realm from the flyout. Due to the variety of configuration possibilities, realm doesn't provide enough information - it would make more sense to include the auth domain, but this is not really feasible to show here. We feel it is better to remove realm from the UI, knowing that all of the auth specifics, when needed, can be queried via API.

Reply:
Perfect, realm is already gone from the flyout :)